conf/modules.config

```
@@ -253,10 +377,19 @@ process {
     withName: SAMTOOLS_FASTQ {
         ext.prefix = { "${meta.id}_${meta.run_accession}.unmapped" }
         publishDir = [
-            path: { "${params.outdir}/samtools/fastq" },
-            mode: params.publish_dir_mode,
-            pattern: '*.fastq.gz',
-            enabled: params.save_hostremoval_unmapped
+            [
+                path: { "${params.outdir}/samtools/fastq" },
+                mode: params.publish_dir_mode,
+                pattern: '*.fastq.gz',
+                enabled: params.save_hostremoval_unmapped
+            ],
+            [
+                path: { "${params.outdir}/analysis_ready_fastqs" },
+                mode: params.publish_dir_mode,
+                pattern: '*.fq.gz',
+                enabled: params.save_analysis_ready_fastqs,
+                saveAs: { ( params.perform_runmerging == false || params.perform_runmerging && !meta.is_multirun ) && params.perform_longread_hostremoval && params.save_analysis_ready_fastqs ? it : null }
```

Are brackets missing here?

```suggestion
                saveAs: { ( params.perform_runmerging == false || ( params.perform_runmerging && !meta.is_multirun) ) && params.perform_longread_hostremoval && params.save_analysis_ready_fastqs ? it : null }
```